squid/.env.example

```diff
@@ -1,4 +1,5 @@
 DB_PORT=5432
+DB_HOST=db
```

Comment:
What is this used for?

Reply:
My intention is to fully use docker to run the apps, with no need our machines to have Nodejs, NPM or any other tool installed beyond Docker.

By default, Squid attempts to connect to the database to `localhost`.

In docker environments, this is not necessary true. You can imagine docker containerization as a separate network layer from your machine, so, that means, when `docker-compose` define the services you want to start, they can be allocated in different IP addresses.

When you run: `docker compose down` and `docker compose up` the IP addresses of the services might be different every time.

In order to connect one service to the other, we use the name we set in `docker-compose.yaml`. As you can see in this Pull Request, I've added two more services, and the postgres database is called `db`, [here is the line of the code related](https://github.com/ChainSafe/Multix/blob/feature/squid-dockerization/squid/docker-compose.yml#L34).

Basically, what I am saying in:
```env
DB_HOST=db
```
Is that squid can locate the database in the service called "db", and docker will make sure to replace "db" to the proper IP address generated by them.

Reply:
Thanks for the explanation. But why is this a variable (I mean an environment variable) this will never change isn't it?

Reply:
This is a variable because it may depends in the way we decide to run the app in local machines, staging and production.

For example, for local environments, you may want to use a Postgres database located in our docker machine, so this value should be: `DB_HOST=db`. But, for production environments, we may want to use AWS RDS (a managed service that provides endpoints of already-served databases with no management needed), in that case the value should be: `DB_HOST=endpoint-provided-by-aws.rds.amazonaws.com`.

Reply:
got it thanks for the explanation.